includes/data.php

```diff
@@ -1547,48 +1572,84 @@ function pods_evaluate_tags( $tags, $sanitize = false, $fallback = null ) {
 		$tags = get_object_vars( $tags );
 
 		// Evaluate array and cast as object.
-		$tags = (object) pods_evaluate_tags( $tags );
+		$tags = (object) pods_evaluate_tags( $tags, $args );
 
 		return $tags;
 	}
 
 	return preg_replace_callback(
 		'/({@(.*?)})/m',
-		function ( $tag ) use ( $sanitize, $fallback ) {
-			return pods_evaluate_tag( $tag, $sanitize, $fallback );
+		function ( $tag ) use ( $args ) {
+			return pods_evaluate_tag( $tag, $args );
 		},
 		(string) $tags
 	);
 }
 
 /**
- * Evaluate tag like magic tag but mapped through pods_v_sanitized.
+ * Evaluate tag like magic tag but sanitized.
  *
  * @since 2.1
  *
  * @param string|array $tag String to be evaluated.
+ * @param array        $args {
+ *     Function arguments.
+ *     @type null|mixed $fallback        The fallback value to use if not set, should already be sanitized.
+ *     @type Pods       $pod             Pod to parse the tags through.
+ *     @type bool       $use_current_pod Whether to auto-detect the current Pod.
+ * }
  *
  * @return string Evaluated content.
  *
  * @see pods_evaluate_tag
  */
-function pods_evaluate_tag_sanitized( $tag ) {
-	return pods_evaluate_tag( $tag, true );
+function pods_evaluate_tag_sanitized( $tag, $args = array() ) {
+	if ( ! is_array( $args ) ) {
+		$args = array();
```

Comment:
:no_entry_sign: Short array syntax must be used to define arrays

<sub>hash: 29f49acb234c7401e5581af4fabb15e7</sub>